Jenkinsfile.release

```diff
@@ -85,9 +85,21 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
             cosa init --branch ${params.STREAM} https://github.com/coreos/fedora-coreos-config
             cosa buildfetch --build=${params.VERSION} \
                 --arch=all --url=s3://${s3_stream_dir}/builds
+            cosa buildprep --ostree --build=${params.VERSION} \
```

Comment:
You can add `--artifact ostree` to the previous `buildfetch` call instead.

Reply:
I'm not sure that works:

```console
$ cosa buildfetch --ostree
usage: cmd-buildfetch [-h] [--url URL] [--stream STREAM] [-b BUILD] [--force] [--arch ARCH] [--artifact ARTIFACT]
cmd-buildfetch: error: unrecognized arguments: --ostree
```